sig-apps/Charter.md

```diff
@@ -0,0 +1,141 @@
+* Do not pick which apps to run on top of Kubernetes
+* Do not recommend one way to do things (e.g., picking a template language)
+
+## Roles
+
+Membership for roles tracked in: [sigs.yaml]
+
+- Chair
+  - Run operations and processes governing the SIG
+  - Seed members established at SIG founding
+  - Chairs *MAY* decide to step down at anytime and propose a replacement.  Use lazy consensus amongst
+    chairs with fallback on majority vote to accept proposal.  This *SHOULD* be supported by a majority of
+    SIG Members.
+  - Chairs *MAY* select additional chairs through a [super-majority] vote amongst chairs.  This
+    *SHOULD* be supported by a majority of SIG Members.
+  - Chairs *MUST* remain active in the role and are automatically removed from the position if they are
+    unresponsive for > 3 months and *MAY* be removed if not proactively working with other chairs to fulfill
+    responsibilities.
+  - Number: 2-3
+  - Defined in [sigs.yaml]
+
+- Subproject Owners
+  - Scoped to a subproject defined in [sigs.yaml]
+  - Seed members established at subproject founding
+  - *MUST* be an escalation point for technical discussions and decisions in the subproject
+  - *MUST* set milestone priorities or delegate this responsibility
+  - *MUST* remain active in the role and are automatically removed from the position if they are unresponsive
+    for > 3 months.
+  - *MAY* be removed if not proactively working with other Subproject Owners to fulfill responsibilities.
+  - *MAY* decide to step down at anytime and propose a replacement.  Use [lazy-consensus] amongst subproject owners
+    with fallback on majority vote to accept proposal.  This *SHOULD* be supported by a majority of subproject
+    contributors (those having some role in the subproject).
+  - *MAY* select additional subproject owners through a [super-majority] vote amongst subproject owners.  This
+    *SHOULD* be supported by a majority of subproject contributors (through [lazy-consensus] with fallback on voting).
+  - Number: Minimum of 3
+  - Defined in [sigs.yaml] [OWNERS] files
+
+- Members
+  - *MUST* maintain health of at least one subproject or the health of the SIG
+  - *MUST* show sustained contributions to at least one subproject or to the SIG
+  - *SHOULD* hold some documented role or responsibility in the SIG and / or at least one subproject
+    (e.g. reviewer, approver, etc)
+  - *MAY* build new functionality for subprojects
+  - *MAY* participate in decision making for the subprojects they hold roles in
+  - Includes all reviewers and approvers in [OWNERS] files for subprojects
+
+## Organizational management
+
+- SIG meets weekly on zoom with agenda in meeting notes
+  - *SHOULD* be facilitated by chairs unless delegated to specific Members
+- SIG overview and deep-dive sessions organized for Kubecon
+  - *SHOULD* be organized by chairs unless delegated to specific Members
+
+- Contributing instructions defined in the SIG CONTRIBUTING.md
```

Comment:
Is there a CONTRIBUTING.md doc?

Reply:
There is. You can [read it here](https://github.com/kubernetes/community/blob/master/sig-apps/CONTRIBUTING.md). Credit @parispittman for getting us to have one.